### ISSUE TITLE
Conda/Windows CI Failure End of August 2022

### ISSUE THREAD

**Issue by traversaro** [robotology/robotology-superbuild#1242]
See https://github.com/robotology/robotology-superbuild/actions/runs/2945594065 . The CI job failed with:
~~~
2022-08-29T05:03:17.2669420Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(83,58): error C2589: '(': illegal token on right side of '::' [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
2022-08-29T05:03:17.2678260Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(83): error C2062: type 'unknown-type' unexpected [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
2022-08-29T05:03:17.2680248Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(83,27): error C2059: syntax error: ')' [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
2022-08-29T05:03:17.2681788Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(86,58): error C2589: '(': illegal token on right side of '::' [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
2022-08-29T05:03:17.2683507Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(86): error C2062: type 'unknown-type' unexpected [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
2022-08-29T05:03:17.2684974Z C:/Miniconda3/envs/test/Library/include/ignition/transport8\ignition/transport/TopicStatistics.hh(86,27): error C2059: syntax error: ')' [D:\a\robotology-superbuild\robotology-superbuild\build\src\icub-basic-demos\demoRedBall\demoRedBall-world.vcxproj] [D:\a\robotology-superbuild\robotology-superbuild\build\icub-basic-demos.vcxproj]
~~~

**Comment by traversaro** [comment]
Probably related to https://github.com/conda-forge/libignition-transport4-feedstock/pull/66 .

**Comment by traversaro** [comment]
Fixed by 8.3.0_1 build of libignition-transport8 (created by https://github.com/conda-forge/libignition-transport4-feedstock/pull/68 that backported https://github.com/gazebosim/gz-transport/pull/350), see https://github.com/robotology/robotology-superbuild/actions/runs/2947489078 .
